src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -2292,4 +2478,42 @@ protected void clear() {
       entries.clear();
     }
   }
+
+  /**
+   * Representation of a source file from an encoded json stream input
+   */
+  private class JsonFileSpec {
+    private final String src;
+    private final String path;
+    private String source_map;
+
+    public JsonFileSpec(String src) {
+      this(src, null, null);
+    }
+
+    public JsonFileSpec(String src, String path) {
```

Comment:
As far as I can tell you're only using this constructor. Why do you need the other two?